miner/worker.go

```diff
@@ -88,6 +89,7 @@ type environment struct {
 	uncles    mapset.Set         // uncle set
 	tcount    int                // tx count in cycle
 	blockSize common.StorageSize // approximate size of tx payload in bytes
+	l1txcount int                // l1 msg count in cycle
```

Comment:
```suggestion
	l1TxCount int                // l1 msg count in cycle
```

Reply:
Fixed in [e416e09](https://github.com/scroll-tech/go-ethereum/pull/350/commits/e416e09aa5c8780e0aa8d0d6b763a887573f0abb).